web/templates/dropdown_widget.hbs

```diff
@@ -1,4 +1,4 @@
-<button id="{{widget_name}}_widget" class="dropdown-widget-button" type="button" {{#if disable_keyboard_focus}}tabindex="-1"{{/if}} name="{{widget_name}}">
+<button id="{{widget_name}}_widget" class="dropdown-widget-button" type="button" {{#if is_setting_disabled}}disabled{{/if}} {{#if disable_keyboard_focus}}tabindex="-1"{{/if}} name="{{widget_name}}">
```

Comment:
This is also a fine way to do, but for other realm settings we disable the settings in JS in `settings_org.maybe_disable_widgets`.

Reply:
I'd be okay with either of the approach, but I find it a bit easier to read directly from the handlebars file that this setting is disabled for non-owners

Reply:
I'm not sure which method is best; the big thing we need to be careful about is live-updating these can be harder when the logic is in the template.